zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/client/ObjectenApiClient.kt

```diff
@@ -50,8 +50,14 @@ class ObjectenApiClient(
         objectypeId: String,
         pageable: Pageable
     ): ObjectsList {
+        val host = if (objecttypesApiUrl.host == "localhost") {
```

Comment:
This is the only way to make it work on for local development. Because in this case, the objecttypesApiUrl is passed to the objectenAPI. Then the objectenAPI tries to get the type with the url. But that fails because the objectenAPI is inside a docker container where the localhost is different.